src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -179,7 +194,9 @@ export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
         })
 
         this.form.submenuResponse.bindPrompter(createRegionSubmenu)
-        this.form.filterPattern.bindPrompter(({ submenuResponse }) => createFilterpatternPrompter(submenuResponse!.data))
+        this.form.filterPattern.bindPrompter(({ submenuResponse }) =>
+            createFilterpatternPrompter(submenuResponse!.data, logGroupInfo ? true : false)
```

Comment:
This works but is somewhat brittle. Just make sure there are wizard tests asserting that `filterPattern` _always_ shows first if `logGroupInfo` is not provided.

Reply:
I believe there is one on line 123 of `searchLogGroup.test.ts`. Lmk if you had something else in mind.